test_runner/regress/test_branching.py

```diff
@@ -172,7 +172,7 @@ def test_cannot_create_endpoint_on_non_uploaded_timeline(neon_env_builder: NeonE
             env.initial_tenant,
             env.initial_timeline,
             MembershipConfiguration(generation=1, members=[sk.safekeeper_id()], new_members=None),
-            int(env.pg_version),
+            int(env.pg_version) * 10000,
```

Comment:
I'm not sure I understand why this changed.

Reply:
The PG version of the environment is `"\d\d"`, not the full PgVersionId that is expected by Safekeeper's CreateTimeline API.